scriptworker/constants.py

```diff
@@ -202,34 +186,41 @@
                 "path_regexes": (
                     r"^(?P<path>/comm-central)(/|$)",
                     r"^(?P<path>/releases/comm-(beta|esr\d+))(/|$)",
-                    r"^(?P<path>/(try-comm-central))(/|$)",
                 ),
             }),),
             'mobile': (frozendict({
                 "schemes": ("https", "ssh", ),
                 "netlocs": ("github.com", ),
                 "path_regexes": tuple([
-                    # XXX We allow Github forks to run staging releases. Please make sure to
```

Comment:
Potential blocking issue: I'm not sure to follow, can we still run staging releases on our personal repos after these changes? If not, what would be the new way to do it?